ADA/ada.qmd

```diff
@@ -36,14 +35,65 @@ The ambition is to enable access to all datasets across DfE and ESFA and retire
 
 ---
 
+# What is Databricks?
+
+Databricks is a browser-based platform developed by Microsoft which allows you to use notebooks to access and analyse data. You can also use it for machine learning. Databricks notebooks support multiple programming languages, including R, Python, and SQL. Unlike RStudio or SQL Server Management Studio (SSMS), you don't have to download any software to be able to use it. You will need to be given access to Databricks and the data you require by the ADA team by completing an "Analytical Data Access Request" service request form in the IT Help Centre. At the moment, not all of the data currently available in SSMS has been migrated to the Delta Lake for use in Databricks. You can check the status of various datasets [in the data catalogue dashboard](https://app.powerbi.com/groups/me/reports/966389db-2e4e-42b6-8416-36aee2fb9503/ReportSection?ctid=fad277c9-c60a-4da1-b5f3-b3b8b34a82f9&experience=power-bi) 
```

Comment:
Is Databricks just about notebooks? I thought it was wider than that? This is a very notebook-focussed initial description?

In my head it's more like a remote desktop / server that you can use to process data. So you can execute code in the cloud against the data in the delta lake? Yes you can write in notebooks in a databricks specific way, though (in my head at least, I may be wrong) that's an option to mention later rather than the core of it?

Wary that going so notebooks heavy in the first paragraphs might make analysts think that it's purely about them?

Reply:
agree we want to be less notebooks heavy, and this is where our guidance can add value over the ADA pages as they only seem to cover the notebooks stuff